src/effects/builtin/flangereffect.cpp

```diff
@@ -222,7 +224,7 @@ void FlangerEffect::processChannel(const ChannelHandle& handle,
         CSAMPLE prevRight = delayRight[framePrev];
         CSAMPLE nextRight = delayRight[frameNext];
 
-        CSAMPLE frac = delayFrames - floorf(delayFrames);
+        auto frac = static_cast<CSAMPLE>(delayFrames - floorf(static_cast<CSAMPLE>(delayFrames)));
```

Comment:
delayFrames can become a float, it is a position, not a CSAMPLE. 

frac is CSAMPLE_GAIN. 
Here we have a case where auto is misleading, basausse only the destination is CSAMPLE_GAIN 

The whole part can be optimized by using modf() including the floor() and ceil() calls above.

Reply:
Please use the suggestion feature or open a new PR. I don't want to refactor code I don't understand.

Reply:
OK, than just remove auto and make delayFrames float.